packages/bot-web-ui/src/components/dashboard/joyrideConfig.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+
+type TCommonJoyrideProprties = Record<'showProgress' | 'showSkipButton' | 'spotlightClicks' | 'disableBeacon', boolean>;
+interface ITourGuideComponent {
+    Header: string | boolean;
+    Para: string;
+}
+
+const TourGuideComponent = ({ Header, Para }: ITourGuideComponent) => {
+    return (
+        <div>
+            <h1>{localize(Header)}</h1>
+            <p>{localize(Para)}</p>
```

Comment:
localize won't work this way. The texts should be explicit

Reply:
done